docs/source/platform/editor-plugins.md

```diff
@@ -94,16 +96,61 @@ module.exports = {
 
 By default, the VS Code extension will look for all files under "./src" to find both the operations and schema definitions for building a complete schema for the application.
 
-<h2 id="performance-insights">Performance insights</h2>
+<h2 id="features">Features</h2>
+
+Apollo for VS Code brings many helpful features for working on a GraphQL project.
+
+<h3 id="syntax">Syntax highlighting</h3>
+
+Apollo's editor extension provides syntax highlighting for all things GraphQL, including schema definitions in `.graphql` files, complex queries in TypeScript, and even client-only schema extensions. Syntax highlighting for GraphQL works out-of-the-box for `.graphql`, `.gql`, `.js` and `.ts` file types!
+
+<h3 id="autocomplete">Intelligent autocomplete</h3>
+
+Once configured, editors have full knowledge of the schema clients are running operations against, including client-only schemas (for things like local state mutations). Because of this, editors have the ability to autocomplete fields and arguments as you type.
+
+<img src="../images/editors/autocomplete.gif" width="80%" style="margin: 5%" alt="vscode completing a field when typing">
```

Comment:
Something about the line highlighting with the intellisense tooltip is making this gif look janky? Does it look like this with another theme?

Reply:
It looks like it's just this theme that does that. Is it bad enough to redo all the screenshots and gifs though? having one with a different theme would be odd